documentation/conf.py

```diff
@@ -110,6 +110,10 @@ def add_ext_to_path():
     "catalog/guides/deploy": "/meta/release_and_deployment/runbooks/catalog.html",
     "catalog/guides/deployment": "/meta/release_and_deployment/index.html#catalog",
     "ingestion_server/guides/deploy": "/meta/release_and_deployment/runbooks/ingestion_server.html",
+    "packages/eslint_plugin/index": "/packages/js/eslint_plugin/index.html",
+    "packages/eslint_plugin/analytics_configuration": "/packages/js/eslint_plugin/analytics_configuration.html",
+    "packages/eslint_plugin/no-unexplained-disabled-test": "/packages/js/eslint_plugin/no-unexplained-disabled-test.html",
+    "packages/openverse_attribution/index": "/packages/python/openverse_attribution/index.html",
```

Comment:
I originally moved these into language-specific sub-paths to accommodate the two `api-client` packages (one `@openverse/api-client` and the other `openverse-api-client`). That's no longer present in _this_ PR, but because we know we want a Python API client, this conflict will happen soon. In that case, I'd rather move these to language specific sub-paths sooner than later, thereby reducing the overall number of redirects we have to add in the future. This would also apply if we created a separate `@openverse/attribution` JS package, for example, and other things of that nature where we need both a JS and Python version.